proposals/mdawson-cpc-charter/CPC-Charter.md

```diff
@@ -44,157 +44,127 @@ direction of the policies set by the Board.
-     from the mentorship program, Project issues and conflicts, opportunities
-     for collaboration between Projects, opportunities for the Foundation in the
-     greater JavaScript community, etc.
+   1. defining and maintaining neutral consensus for the technical vision for the _$FOUNDATION_.
+   1. creating conceptual architecture for how the projects fit together.
+   1. Voting members are responsible for approving new projects within the scope
+      for _$FOUNDATION_ set by the Governing Board, as outlined in [XXXX].
+   1. Voting members are responsible for making final decisions on aligning projects,
+      removing or archiving projects, as outlined in [XXXX].
+   1. Voting members are responsible for approving funds for budgets delegated 
+      to the project. A good example is the travel fund.
+   1. follow and be up to date on Board/$FOUNDATION initiatives and communicate them to the projects.
+   1. defining common practices to be implemented across _$FOUNDATION_ projects, if any.
+   1. The CPC is also responsible for mediating technical conflicts between
+      _$FOUNDATION_ Projects when attempts to resolve those conflicts within the Project
+      were unsuccessful. 
+   1. Creating a yearly budget that can be drawn on by the projects.
+   1. The CPC serves as the _$FOUNDATION_'s primary technical
+      liaison body with external open source projects, consortiums and groups and is
+      also responsible for ensuring that the _$FOUNDATION_ has appropriate technical
+      participation in standards bodies by finding an encouraging projects members to
+      particiapte in these bodies.
+   1. Managing the Individual membership program.
+   1. Managing programs to improve inclusivity and diversity.
+   1. Guiding projects into appropriate Foundation paths.
+   1. Shared resources, tooling and funding for projects and Commons.js (Events
+      Collaborator Summit, Travel fund).
+   1. Code of Conduct incident response support.
+   1. Developing a framework for community end-user feedback.
+   1. Managing programs to offer mentorship to external individuals.
+   1. Managing developer outreach programs.
+   1. Supporting project advocacy and recognition programs.
+   1. Identifing, recruiting and engaging prospective projects.
+   1. Voting members are responsible for making a final decision if consensus
+      cannot be reached among members.
+   1. Members of the CPC, or committees consisting of subsets of the CPC
+      membership will be expected to meet on a regular basis to discuss topics
+      such as new project acceptance into the mentorship program, project graduation
+      from the mentorship program, Project issues and conflicts, opportunities
+      for collaboration between Projects, opportunities for the Foundation in the
+      greater JavaScript community, etc.
+   1. When needed, members of the CPC will be expected to create subcommittees
```

Comment:
This should be the first point imho... a bunch of my prior notes may be resolved by this

Reply:
Will move, makes sense to me